test/e2e/support/utils/is-current-url.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Node dependencies
+ */
+import { URL } from 'url';
+
+/**
+ * Internal dependencies
+ */
+import { createURL } from './create-url';
+
+/**
+ * Checks if current URL is a WordPress path.
+ *
+ * @param {string} WPPath String to be serialized as pathname.
+ * @param {?string} query String to be serialized as query portion of URL.
+ * @return {boolean} Boolean represents whether current URL is or not a WordPress path.
+ */
+export function isCurrentURL( WPPath, query = '' ) {
```

Comment:
Would be cool to see all these query arguments take objects instead, like the admin function I commented on. That way we can just pass JS objects to them instead of building the string ourselves.